152_data_reading_program/sensor_reader.py

```diff
@@ -0,0 +1,85 @@
+# This script is used to gather data from the SCD-30 sensor attached to
+# the MCP-2221. This script is influenced by Carter Nelson and
+# Bryan Siepert's scripts written for Adafruit Industries in 2020 and 2021
+# It enhances these scripts by calling the linux export statement directly,
+# By presenting the data much more cleanly
+# by having a discrete function to reset the SCD and calling this function
+# when necessary when the SCD-30 crashes (which happens sometimes)
+# and also by recovering from failure to read from the SCD-30
+# by trying again next time instead of just crashing altogether.
+# Unfortunately, there is still a problem where the sensor stops working
+# eventually and must be unplugged from the computer and plugged back in
+# again to make it work.
+# @author Greg Ross, version 1, October 29,2021
+
+# According to Carter'script, the following commented line causes
+# prevents an issue with the reset on MCP-2221
+# pylint: disable=protected-access
+
+import sys # For force quit.
+import os # To do the system call to set export
+# Set the export statment command with a system call
+# in case it is not already set.
+# If this system call is not made, import board will fail
+os.system("export BLINKA_MCP2221=1")
+import time # For sleep function
+import adafruit_scd30 # Adafruit's Library for the SCD-30
+import busio # For I2C
+import board # For MCP-2221
```

Comment:
Generally you want to put all imports first, starting from stdlib modules, then third-party modules and other modules from the project, each group separated by a blank line.  I also like to order the imports from shorter to longer, i.e.:
```py
import os
import sys
import time

import board
import busio
import adafruit_scd30
```

There is no need to comment what you imported the module for, since it's easy to see with a ctrl+f and might change over time making the comment obsolete.

If `import board` doesn't work without the `export` then put both at the end, one next to the other.